src/stores/Checkout.js

```diff
@@ -472,6 +473,36 @@ class CheckoutStore {
     }
   });
 
+  EntitlementClaim = flow(function * ({entitlementSignature}) {
+    const decode = yield this.client.DecodeSignedMessageJSON({signedMessage: entitlementSignature});
+    decode?.message && rootStore.log("EntitlementClaim msg", decode.message);
+    const tenant_id = decode?.message?.tenant_id;
+
+    const tok = this.walletClient.AuthToken();
+    const url = UrlJoin(this.walletClient.client.authServiceURIs[0], "wlt", "act", tenant_id);
+    const options = {
+      method: "POST",
+      headers: {"Authorization": "Bearer " + tok},
+      body: JSON.stringify({"op":"nft-claim-entitlement", "signature": entitlementSignature}),
+    };
+
+    let resp = "";
+    yield fetch(url, options)
+      .then(response => response.json())
+      .then(data => {
+        rootStore.log("Ok:", data);
+        // get pid_123 from "nft-claim-entitlement:iq__...:sku...:pid_123[:dup_8tTXKSML]"
+        resp = data.op?.split(":")[3];
+      })
+      .catch(error => {
+        rootStore.error("Error:", error);
+        resp = JSON.stringify(error);
+      });
+
+    rootStore.log("EntitlementClaim", "url", url, "options", options, "resp", resp);
```

Comment:
still using redundant logging methods, use `rootStore.Log` and string interpolation (\`something ${var} etc\`)

Reply:
that doesn't allow you to use the built-in json support in the console.. you don't like that?

Reply:
if I'm adding logging for debugging purposes when developing I'll just use console.log, and I generally don't like leaving debug logging in, only error reporting. 

it can definitely be improved, and in the creator studio I have a [better system with log levels](https://github.com/eluv-io/elv-creator-studio/blob/a73d07146098edacddfa8f08a9aeb80015cf04b1/src/stores/index.js#L138), but at this point in this app, rootStore.Log is mainly so I don't have to add eslint exceptions for using console.error when logging errors.

Reply:
i'm talking about using proper varargs, which i added

it has nothing to do w/ eslint, that was inside a rootStore.log()

Reply:
I'm not opposed to the new log methods, but we should also change the other ~100 instances of rootStore.Log to use them as well if we want to switch

Reply:
ah good idea... i would have been happy to do that, maybe next time